internal/git/gitsync_processor.go

```diff
@@ -156,16 +190,29 @@ func watchRepo(ctx context.Context, r *git.Repository, gitSync *v1alpha1.GitSync
 			return err
 		}
 	} else {
-		ticker := time.NewTicker(timeInterval * time.Minute)
+		ticker := time.NewTicker(10 * time.Second)
```

Comment:
did this accidentally get changed for testing maybe?

Reply:
ohh, my bad!! Let me revert it